src/modules/src/supervisor.c

```diff
@@ -297,6 +297,8 @@ static void postTransitionActions(SupervisorMem_t* this, const supervisorState_t
   }
 }
 
+uint8_t tumbleCheckEnabled = 1;
```

Comment:
I would prefer to keep the `SUPERVISOR_TUMBLE_CHECK_ENABLE` in the platform defaults. We generally aim to keep anything configurable out of the source files (other than platform defaults), and this option can be valuable for those flying rate controllers (doing flips) or making extreme hardware mods.

Reply:
No problem, putting it back